src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -73,7 +73,7 @@
                                           'relevant for listed add-ons.') }}">?</span>
   </li>
 {% endif %}
-{% if addon.type == amo.ADDON_EXTENSION and amo.FIREFOX in addon.compatible_apps %}
+{% if addon.is_listed and addon.type == amo.ADDON_EXTENSION and amo.FIREFOX in addon.compatible_apps %}
```

Comment:
similar to above, has there been a decision to only check listed versions for compatibility?